funnel/templates/project.html.jinja2

```diff
@@ -99,78 +131,57 @@
-                        <div class="card__title__heading session-card__header__title">
-                          {% if session.speaker %}<h4 class="mui--text-subhead mui--text-bold zero-bottom-margin zero-top-margin">{{ session.speaker }}</h4>{% endif %}
-                          <h3 class="mui--text-subhead zero-bottom-margin zero-top-margin">{{ session.title }}</h3>
-                        </div>
-                      {% if session.proposal %}</a>{% else %}</div>{% endif %}
-                    {% else %}
-                      {% if session.proposal %}
-                        <a class="card__header clickable-card" href="{{ session.proposal.url_for() }}" title="{{ session.title }}" aria-label="{{ session.title }}">
-                      {% else %}
-                        <div class="card__header">
+                      {% if session.description %}
+                        {% if session.proposal %}
+                          <a class="card__body clickable-card" href="{{ session.proposal.url_for() }}" title="{{ session.title }}" aria-label="{{ session.title }}">
+                        {% else %}
+                          <div class="card__body">
+                        {% endif %}
+                          <div class="mui--text-body1 mui--text-light truncate">{{ session.description|striptags }}</div>
+                        {% if session.proposal %}</a>{% else %}</div>{% endif %}
                       {% endif %}
-                        <div class="card__header__title">
-                          <h3 class="mui--text-title card__title__heading mui--text-bold">{{ session.title }}</h3>
-                          {% if session.speaker %}
-                            <h4 class="mui--text-subhead card__title__heading">by {{ session.speaker }}</h4>
-                          {% endif %}
-                        </div>
-                      {% if session.proposal %}</a>{% else %}</div>{% endif %}
-                    {% endif %}
-                    {% if session.proposal %}
-                      <a class="card__body clickable-card" href="{{ session.proposal.url_for() }}" title="{{ session.title }}" aria-label="{{ session.title }}">
-                    {% else %}
-                      <div class="card__body">
-                    {% endif %}
-                      <div class="mui--text-body1 mui--text-light truncate">{{ session.description|striptags }}</div>
-                    {% if session.proposal %}</a>{% else %}</div>{% endif %}
-                  </div>
-                </li>
-              {% endif %}
-            {%- endfor -%}
-          </ul>
-          {%- if project.schedule_state.PUBLISHED %}<p class="mui--text-left"><a class="mui-btn mui-btn--raised mui-btn--accent" href="{{ project.url_for('schedule') }}" data-action="View schedule(page)">{% trans %}View Schedule{% endtrans %}</a></p>{%- endif %}
```

Comment:
What is `data-action="View schedule(page)"`? If it's a UI string, it needs `{% trans %}` and a space before `(`.

Reply:
data-action is not user facing text, it is for event tracking

Reply:
Okay.